src/streams/StreamList.js

```diff
@@ -72,7 +72,7 @@ export default class StreamList extends PureComponent<Props> {
             unreadCount={unreadByStream[item.stream_id]}
             isMuted={item.in_home_view === false} // if 'undefined' is not muted
             showSwitch={showSwitch}
-            isSwitchedOn={item.subscribed}
+            isSubscribed={item.subscribed}
```

Comment:
Hmmm, interesting, thanks for catching that!

This is unfortunately sort of annoying to fix, since we've written `PseudoSubscription` as a enum, which is I think not actually what we want here. See [#mobile-team > Stream/Subscription split](https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/Stream.2FSubscription.20split) for my thoughts on the right approach to fixing that — but I'd prefer for that to be in a different PR, since I think it's somewhat involved.